index.html

```diff
@@ -2,17 +2,212 @@
+          </div>
+          <div class="main-subject">
+            <p>Red Carpet</p>
+          </div>
+          <div class="main-content">
+            <p>
+              The traditional ceremonial route for celebrities and guest to showcase there glitz and glamour ahead of
+              the event.
+            </p>
+          </div>
+        </div>
+        <div class="main-program-content flex-center">
+          <div class="main-program-img">
+            <img src="https://img.icons8.com/emoji/48/000000/microphone-emoji.png" alt="presentation" />
+          </div>
+          <div class="main-subject">
+            <p>Presentation</p>
+          </div>
+          <div class="main-content">
+            <p>
+              Host of the event gives a short speech about her journey til date, inspirations, what is about to be
+              experienced and what to come.
+            </p>
+          </div>
+        </div>
+        <div class="main-program-content flex-center">
+          <div class="main-program-img">
+            <img src="https://img.icons8.com/color/50/000000/fashion-model.png" alt="runway model" />
+          </div>
+          <div class="main-subject">
+            <p>Fashion Runway</p>
+          </div>
+          <div class="main-content">
+            <p>
+              Our models showcase our different line and experties and glamour for all weather on the runway.
+            </p>
+          </div>
+        </div>
+        <div class="main-program-content flex-center">
+          <div class="main-program-img">
+            <img src="https://img.icons8.com/color/48/000000/people-working-together.png" alt="get together" />
+          </div>
+          <div class="main-subject">
+            <p>Get Together</p>
+          </div>
+          <div class="main-content">
+            <p>
+              Our guest meet the host and other fashion celebrities to know each other and merry.
+            </p>
+          </div>
+        </div>
+        <div class="main-program-content flex-center">
+          <div class="main-program-img">
+            <img src="https://img.icons8.com/dusk/64/000000/donate.png" alt="donate" />
+          </div>
+          <div class="main-subject">
+            <p>Fund-raiser</p>
+          </div>
+          <div class="main-content">
+            <p>
+              The platform is willing to help young and talented fashion designer, to showcase what they can do
+              on a large scale.
+            </p>
+          </div>
+        </div>
+      </div>
+      <div class="see-more">
+        <a href="#">SEE MORE</a>
+      </div>
+    </div>
+
+    <div class="get-ticket-btn flex-center">
+      <button type="submit">Get Tickets</button>
+    </div>
+  </section>
+
+  <section class="feature">
+    <div class="feature-speaker">
+      <div class="flex flex-center">
+        <p class="feature-title">Celebrity Guests</p>
+        <div class="feature-title-line"></div>
+      </div>
```

Comment:
- [ ] The underline text should be given a higher margin 
![image](https://user-images.githubusercontent.com/33067256/128328012-be6c2beb-8cca-4fa9-b8c3-abc3fc891ca1.png)

Reply:
it has been adjusted. 🙏🏽